.env.example

```diff
@@ -130,3 +130,7 @@ SOURCE_CODE_REPOSITORY_LINK='https://github.com/MIT-LCP/physionet-build'
 # These variables are used to set the theme colors for the site
 DARK='#343A40'
 PRIMARY='#002A5C'
+
+# Users settings
+# maximum number of emails that can be associated to a user model
+ASSOCIATED_EMAILS_ALLOWED =  10
```

Comment:
Instead of `ASSOCIATED_EMAILS_ALLOWED`, how about something like `MAX_ASSOCIATED_EMAILS`? Nitpick but there is an extra space before the `10`

Reply:
Thank you. That's a way better name